src/MoBi.Presentation/Tasks/ObservedDataTask.cs

```diff
@@ -158,16 +159,30 @@ private static MoBiMacroCommand deleteAllResultsFromSimulationCommand(IMoBiSimul
          return macroCommand;
       }
 
-      private DimensionInfo.DataImporterSettings createDataImportSettings()
+      private (IReadOnlyList<MetaDataCategory>, DataImporterSettings) initializeSettings()
       {
-         var settings = new DimensionInfo.DataImporterSettings
+         var dataImporterSettings = new DataImporterSettings
          {
             IconName = ApplicationIcons.MoBi.IconName,
             Caption = $"{AppConstants.PRODUCT_NAME} - {AppConstants.Captions.ImportObservedData}"
          };
-         addNamingPatterns(settings);
-         settings.NameOfMetaDataHoldingMolecularWeightInformation = AppConstants.Parameters.MOLECULAR_WEIGHT;
```

Comment:
This was using the wrong constant! Name of the parameters instead of the ObservedDataConstant defined.